ch.elexis.core.data.tests/src/ch/elexis/data/Test_VkPreise.java

```diff
@@ -0,0 +1,69 @@
+package ch.elexis.data;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.UUID;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import ch.rgw.tools.JdbcLink;
+import ch.rgw.tools.JdbcLinkException;
+import junit.framework.Assert;
+	
+public class Test_VkPreise extends AbstractPersistentObjectTest {
+	
+	public Test_VkPreise(JdbcLink link){
+		super(link);
+	}
+
+
+	@Test
+	public void testStoringVkPreise	() throws SQLException{
+		String id = String.valueOf(UUID.randomUUID()).substring(0, 20);
+		link.exec("INSERT INTO VK_PREISE (ID) VALUES (" + JdbcLink.wrap(id)  +")");
+		ResultSet res = link.getStatement().query("SELECT id FROM VK_PREISE WHERE ID=" + JdbcLink.wrap(id));
```

Comment:
Der Code in Elexis, der dies durchführt findet sich in ch.elexis.data.VerrechenbarAdapter#setVKMultiplikator bzw. getMultiplikator. Könnten wir die Funktionalität als Test über die entsprechenden Codes integrieren?